### ISSUE TITLE
[Bug]: Incorrect command when unexpected causes program to crash

### ISSUE THREAD

**Issue by Masamune3210** [ThioJoe/YT-Spammer-Purge#473]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

After excluding users from the process, if the user types exclude again after to add more they noticed, the program errors out with Unknown Error - Code Z-3. Program should preferably allow more exclusions to be added, or at least handle the incorrect command with a notice like in other places in the program

### Release version

2.14.2

### Steps to reproduce

1. Scan video for comments
2. type exclude
3. add users
4. notice more users that need to be excluded and try to type exclude again to add them to the exclusions
5. crash

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):
File "YTSpammerPurge.py", line 1513, in <module>
File "YTSpammerPurge.py", line 1490, in main
File "YTSpammerPurge.py", line 1346, in primaryInstance
File "Scripts\operations.py", line 803, in exclude_authors
AttributeError: 'NoneType' object has no attribute 'group'

Unknown Error - Code: Z-3
```

### Screenshots

![image](https://user-images.githubusercontent.com/1053504/152098894-d5e16a8a-40e1-46f3-9fc0-6bd8737159ec.png)

**Comment by Masamune3210** [comment]
Absolutely love the idea btw! The fact that this isn't part of YouTube proper is a crime

**Comment by TheMythologist** [comment]
Error boils down to `re.search()` not matching anything and thus returning `None`, causing `None.group(0)` to return this error. A simple error checking to see if `re.search()` returns `None` before continuing the execution process will fix this (something similar to this:
https://github.com/ThioJoe/YT-Spammer-Purge/blob/887460452f6834d973480e502388fb0413319c4b/Scripts/community_downloader.py#L28-L30

**Comment by ThioJoe** [comment]
Actually you can technically exclude multiple times in a row, the error you encountered was just because you typed "exclude" without putting any numbers.

One thing to note (which I just realized and will have to fix), is that while you can exclude multiple times in a row, the log file will only list at the bottom the authors you excluded last, not the first time. Though it will still remove all of their comments from the top list of comments to be removed, so it's more of a cosmetic thing.

**Comment by ThioJoe** [comment]
Should be fixed with release 2.14.3 and https://github.com/ThioJoe/YT-Spammer-Purge/commit/24922d94c09647de606918ef447fa1422f26f91e
